packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -122,12 +122,8 @@ export default ({ api, coreSagas, networks }) => {
           yield put(stopSubmit(LOGIN_FORM))
           break
         // web - institutional exchange login
-        case userType === 'institutional' && institutionalPortalEnabled:
-          window.open(
-            `${institutionalDomain}/institutional/portal/?jwt=${jwtToken}`,
-            '_self',
-            'noreferrer'
-          )
+        case userType === 'institutional' && institutionalPortalEnabled && redirect:
+          window.open(`${redirect}?jwt=${jwtToken}`, '_self', 'noreferrer')
```

Comment:
what if there is no redirect? should we leave the fallback?

Reply:
[this ticket](https://blockchain.atlassian.net/browse/FWN-1345) says that only users coming from .com/institutional should be directed to the institutional app after login. All other users who just click the footer ‘Institutional User’ in the login page should just go to the regular exchange app. So if there's no redirect, it should go to the default case and open the regular exchange app. I can add a comment to make it clear.